### PR TITLE
fixed issue with findElements method for elements in the specific state

### PR DESCRIPTION
**Pull request by DmitryBogatko**
updated test because of changes
additionally added QuickStartExample and reference to it from README file
pom updated with list of developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ We use interfaces where is possible, so you can implement your own version of ta
...
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </repository>
-        <repository>
-            <id>ossrh</id>
-            <url>https://repo1.maven.org/maven2/</url>
-        </repository>
-    </repositories>
+    <repository>
+        <id>ossrh</id>
+        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (mialeska) [inline README.md]: these repositories aren't really needed

File: README.md [hunk 2]
```diff
@@ -39,24 +39,28 @@ Browser browser = BrowserManager.getBrowser();
...
-ITextBox txbEmail = new ElementFactory().getTextBox(By.id("email_create"), "Email");
+ElementFactory elementFactory = new ElementFactory();
+ITextBox txbSearch = elementFactory.getTextBox(By.id("searchInput"), "Search");
+
 ```
 
 5. Call element's methods to perform action with element: 
 ```java
-txbEmail.type("email@domain.com");
+txbSearch.type("quality assurance");
```
Comment (mialeska) [inline README.md]: you can submit the search here and wait for page to load)

File: src/main/resources/localization/en.json [hunk 3]
```diff
@@ -63,5 +63,6 @@
   "loc.waitnotexists" : "Wait until element does not exist in DOM during %1$s seconds",
   "loc.no.elements.found.in.state" : "no elements with locator '%1$s' found in state '%2$s' during %3$s seconds",
   "loc.no.elements.found.by.locator" : "No elements were found by locator '%1$s'",
-  "loc.elements.were.found.but.not.in.state" :  "Elements were found by locator '%1$s'. But %2$s"
+  "loc.elements.were.found.but.not.in.state" :  "Elements were found by locator '%1$s'. But %2$s",
```
Comment (mialeska) [inline src/main/resources/localization/en.json]: not sure that ". But desiredState" is very informative. can we additionally say "were not in state"?

File: src/main/resources/localization/ru.json [hunk 4]
```diff
@@ -63,5 +63,6 @@
   "loc.waitnotexists" : "Ожидаем исчезновения элемента из DOM в течении %1$s",
   "loc.no.elements.found.in.state" : "не удалось найти элементов по локатору '%1$s' в состоянии '%2$s' на протяжении %3$s секунд",
   "loc.no.elements.found.by.locator" : "Не удалось найти элементов по локатору '%1$s'",
-  "loc.elements.were.found.but.not.in.state" :  "Удалось найти элементы по локатору '%1$s'. Но %2$s"
+  "loc.elements.were.found.but.not.in.state" :  "Удалось найти элементы по локатору '%1$s'. Но %2$s",
```
Comment (mialeska) [inline src/main/resources/localization/ru.json]: same as in English variant

**Comment by mialeska** [comment]
seems that #53 should be closed then

File: README.md (hunk 1 above)
Comment (DmitryBogatko) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (DmitryBogatko) [reply README.md]: fixed

File: src/main/resources/localization/en.json (hunk 3 above)
Comment (DmitryBogatko) [reply src/main/resources/localization/en.json]: fixed

File: src/main/resources/localization/ru.json (hunk 4 above)
Comment (DmitryBogatko) [reply src/main/resources/localization/ru.json]: fixed

File: src/test/java/aquality/selenium/utils/ElementActionRetrierTests.java [hunk 5]
```diff
@@ -57,16 +68,16 @@ public void testRetrierShouldThrowUnhandledException() {
 
     @Test(dataProvider = "handledExceptions")
     public void testRetrierShouldWorkCorrectTimes(RuntimeException handledException) {
-        Date startTime = new Date();
+        AtomicInteger actualAttempts = new AtomicInteger(0);
         try {
             ElementActionRetrier.doWithRetry(() -> {
+                Logger.getInstance().info("current attempt is "+actualAttempts.incrementAndGet());
```
Comment (DmitryBogatko) [inline src/test/java/aquality/selenium/utils/ElementActionRetrierTests.java]: please, add spaces between symbols '"+'

Comment (mialeska) [reply src/test/java/aquality/selenium/utils/ElementActionRetrierTests.java]: fixed

**Pull request merged**